src/model/DataDefinitions.ts

```diff
@@ -1255,172 +1180,456 @@ export const A_LINE_BOX: ContainerTemplate = {
+      { ...MISSING_FIELDS, name: "Chloroprep", size: "Small", quantity: 2 },
+      {
+        ...MISSING_FIELDS,
+        name: "Needle & stabiliser dressing",
+        size: "15mm : Pink",
+        quantity: 2,
+      },
+      {
+        ...MISSING_FIELDS,
+        name: "Needle & stabiliser dressing",
+        size: "25mm : Blue",
+        quantity: 2,
+      },
+      {
+        ...MISSING_FIELDS,
+        name: "Needle & stabiliser dressing",
+        size: "45mm : Yellow",
+        quantity: 2,
+      },
+      { ...MISSING_FIELDS, name: "3 way tap" },
+      { ...MISSING_FIELDS, name: "Syringe 50ml (luer lock)" },
+    ],
+  };
+
+export const BLACK_BAG_MAIN_COMPARTMENT_ACCESS_IV: PartialFixedContainerTemplate =
+  {
+    name: "Access IV (Main Compartment)",
+    containerType: "Compartment",
+    items: [
+      { ...MISSING_FIELDS, name: "Disposable tourniquet", quantity: 2 },
+      { ...MISSING_FIELDS, name: "Chloroprep", size: "Small", quantity: 4 },
+      { ...MISSING_FIELDS, name: "Cannula dressing", quantity: 2 },
+      ITEM_GAUZE_SWABS,
+      { ...MISSING_FIELDS, name: "Tape" },
+      { ...MISSING_FIELDS, name: "Cannula", size: "24G : Yellow", quantity: 2 },
+      { ...MISSING_FIELDS, name: "Cannula", size: "22G : Blue", quantity: 2 },
+      { ...MISSING_FIELDS, name: "Cannula", size: "20G : Pink", quantity: 2 },
+      { ...MISSING_FIELDS, name: "Cannula", size: "18G : Green", quantity: 2 },
+      { ...MISSING_FIELDS, name: "Cannula", size: "16G : Grey", quantity: 2 },
+      { ...MISSING_FIELDS, name: "Cannula", size: "14G : Orange", quantity: 2 },
+    ],
+  };
+
+export const BLACK_BAG_MAIN_COMPARTMENT_BOTTOM_SECTION: PartialFixedContainerTemplate =
+  {
+    name: "Bottom Section (Main Compartment)",
+    containerType: "Compartment",
+    items: [
+      { ...MISSING_FIELDS, name: "Adult HFM" },
+      { ...MISSING_FIELDS, name: "Paediatric HFM" },
+      { ...MISSING_FIELDS, name: "Nasal Cannula" },
+      { ...MISSING_FIELDS, name: "Mask", size: "1" },
+      { ...MISSING_FIELDS, name: "Mask", size: "2" },
+      { ...MISSING_FIELDS, name: "Mask", size: "3" },
+      { ...MISSING_FIELDS, name: "Mask", size: "4" },
+      { ...MISSING_FIELDS, name: "Mask", size: "5" },
+      { ...MISSING_FIELDS, name: "Adult BVM with clear mask" },
+    ],
+  };
+
+export const BLACK_BAG_LEFT_POCKET: PartialFixedContainerTemplate = {
+  name: "Left Side Pocket",
+  containerType: "Pocket",
   items: [
+    { ...MISSING_FIELDS, name: "Pelvic Binder (Prometheus)" },
+    { ...MISSING_FIELDS, name: "Adult C Spine Collar" },
+    { ...MISSING_FIELDS, name: "Paeds C Spine Collar" },
+  ],
+};
+
+export const BLACK_BAG_RIGHT_POCKET: PartialFixedContainerTemplate = {
+  name: "Right Side Pocket",
+  containerType: "Pocket",
+  items: [{ ...MISSING_FIELDS, name: "Paeds BVM with clear mask" }],
+};
+
+export const BLACK_BAG_1: StorageAreaTemplate = {
+  storageAreaId: "black-bag-1",
+  name: "Black Bag (First Responder) 1",
```

Comment:
Should this be "Medic 1 - Black Bag 1" ?

Reply:
Yes, thanks - have renamed both